metastats-domain/src/test/groovy/life/qbic/metastats/request/RequestExperimentDataSpecification.groovy

```diff
@@ -46,7 +47,7 @@ class RequestExperimentDataSpecification extends Specification {
 
         then:
         !res1
-        !res2
+        assert !res2
```

Comment:
As far as i understood it you shouldn't need the assert statement explicitly in the then statement of a spock based test: See 
https://dzone.com/articles/testing-your-code-with-spock
```suggestion
        !res2
```